src/app/submission/components/metadata-list/metadata-list.component.ts

```diff
@@ -203,6 +204,10 @@ export class MetadataListComponent implements OnInit, OnDestroy {
   }
 
   openDialog(rowIndex: number): void {
+    if (!this.isEditable) {
+      return;
+    }
```

Comment:
Am I right that this is "just-in-case", since the button will already be disabled?

Reply:
yeah, i did it this way first but didnt see the harm in leaving it